curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/5dfb6250eacea3f48c6300b2.md

```diff
@@ -84,7 +83,7 @@ assert.notMatch(code, /\<img\s+.+\s+src\s*=\s*https:\/\/cdn\.freecodecamp\.org\/
           <li>laser pointers</li>
           <li>lasagna</li>
         </ul>
-        
+
```

Comment:
please do not remove the indentation, the element added will be sibling of `ul`, so the indentation is in the right place